docs/content/Contribution guidelines/operations/packaging-and-versioning.md

```diff
@@ -77,22 +77,22 @@ This Helm chart describes a deployment of the KubeStellar core
 components (a space provider, KubeStellar controllers, initialization) as
 workload for a Kubernetes cluster.
 
-The source of this Helm chart is in the `core-helm-chart` subdirectory
```

Comment:
This document is obsolete. Updating it for this one change would be confusing. See docs/content/v02.0/packaging.md.

Reply:
change undone